src/web/components/ApiKeyManagement/KeyTable.tsx

```diff
@@ -0,0 +1,51 @@
+import { ApiKeyDTO } from '../../../api/services/adminServiceHelpers';
+import { TableNoDataPlaceholder } from '../Core/TableNoDataPlaceholder';
+import { Tooltip } from '../Core/Tooltip';
+import KeyItem from './KeyItem';
+
+import './KeyTable.scss';
+
+function NoKeys() {
+  return (
+    <TableNoDataPlaceholder
+      icon={<img src='/group-icon.svg' alt='group-icon' />}
```

Comment:
I think it'll be a rare case where we display the empty table, but we may eventually want to change this icon. No action needed for now, though.

Reply:
Yeah that was my thinking. It seemed better to have something here then nothing.